app/controller/articles-controller.js

```diff
@@ -11,3 +11,13 @@ exports.getArticle = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getArticles = (req, res, next) => {
+  selectArticles()
+    .then((articles) => {
+      res.status(200).send({ articles: articles });
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
```

Comment:
Controller looks good - if you feel like it remember when you're putting a variable on a key with the same name you can use shorthand
`{ articles: articles }` => `{ articles }`